src/hooks/apis/useCloudRegions.js

```diff
@@ -0,0 +1,26 @@
+import { useQuery } from 'react-query';
+
+import apiRequest from '../../services/apiRequest';
+import { getQueryString } from '../../utils/queryString';
+
+export const queryKey = (query) => ['cloud_regions', query];
+
+const getCloudRegions = async ({ provider, instanceType }) => {
+  const queryString = getQueryString({ instance_type: instanceType });
+  const { data } = await apiRequest.get(
+    `/api/rhacs/v1/cloud_providers/${provider}/regions?${queryString}`
+  );
+  return data;
+};
+
+export const useCloudRegions = (query, options) => {
+  if (!query.provider && options.enabled !== false) {
+    console.error('useCloudRegions: provider is required');
```

Comment:
Are you sure it's ok to print the error to the console?

Reply:
Yeah I'm gonna remove this, it's a bit useless